src/libraries/System.Private.CoreLib/src/System/Threading/ManualResetEventSlim.cs

```diff
@@ -186,6 +186,7 @@ public ManualResetEventSlim(bool initialState, int spinCount)
         /// </summary>
         /// <param name="initialState">Whether the event is set initially or not.</param>
         /// <param name="spinCount">The spin count that decides when the event will block.</param>
+#pragma warning disable IDE0060 // Remove unused parameter spinCount, on single-threaded systems, the spin count is not used.
```

Comment:
curious why this triggers, it seems to be used? unless the analyzer notices that Environment.IsSingleProcessor is always true?

Reply:
It notices and that's a good thing :)
It tells me to propagate `const` further, bu I don't want to pollute the whole codebase